src/stack_manager.py

```diff
@@ -136,8 +138,10 @@ def DrawStack(self, scope: Scope) -> Image:
 
         for idx in reversed(deferred):
             child = images[idx]
-            child.tag.size = (images[idx+1].size[0], child.tag.size[1]) # tuple doesn't support assignment
-            images[idx] = self.drawing_manager.DrawTextMeme(child.tag, child.scoped_tags, child.children)
+            # tuple doesn't support assignment
+            child.tag.size = (images[idx+1].size[0], child.tag.size[1])
```

Comment:
Are we happy with this or do we want to generalize it at all?
like,
```python
child.tag.size = (images[idx + 1].size[0] if child.tag.size[0] == 'lookahead' else child.tag.size[0]
                  images[idx + 1].size[1] if child.tag.size[1] == 'lookahead' else child.tag.size[1])```

Reply:
Alternatively, modify it even further to allow bottom text?

Reply:
uhhhh idk?

Reply:
Thinking a little bit more, I'm going to say no on bottomtext, if only because the deferring logic will have to get more complicated, and I think there's no point in supporting "left text" at the moment, so while its worth keeping in mind, I don't think we want to make this change atm